packages/mongo/README.md

```diff
@@ -36,6 +36,11 @@ const keyv = new Keyv({
 })
 ```
 
+```
+NOTE:
+The mongo adapter deviates from the adapter in terms of uri options, you need to specify url instead of uri when passing the DB URL in the options 
```

Comment:
is this note still relevant?

Looks like Mongo constructor is using `uri, options` interface as well:

https://mongodb.github.io/node-mongodb-native/4.4/classes/MongoClient.html#constructor

Reply:
Yep, the mongo constructor needs `{ url }` as opposed to `{ uri }` for everything else